<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>730</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageStart">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="btnNew">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Create New</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="txtName"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lblName">
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>What is your name?</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="btnSelecExist">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Select Existing Event</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageNewDate">
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="3" column="1">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Select A Time</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="btnNewDateNext">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2" colspan="3">
           <widget class="QLineEdit" name="eventName">
            <property name="toolTip">
             <string>This box serves to provide a name for your event.</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Name Your Event</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="btnNewDateBack">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QCalendarWidget" name="calendarWidget">
            <property name="selectedDate">
             <date>
              <year>2017</year>
              <month>9</month>
              <day>17</day>
             </date>
            </property>
            <property name="minimumDate">
             <date>
              <year>2017</year>
              <month>9</month>
              <day>17</day>
             </date>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageNewTime">
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="4">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="5">
           <widget class="QPushButton" name="btnNewTimeToggle">
            <property name="text">
             <string>24 Hour Toggle</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="5">
           <widget class="QPushButton" name="btnNewTimeSave">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="5">
           <widget class="QPushButton" name="btnNewTimeBack">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Select Event Times</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>1106</width>
               <height>568</height>
               <width>22</width>
               <height>22</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_17"/>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageReturn">
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="1">
           <widget class="QPushButton" name="btnReturnToStart">
            <property name="text">
             <string>Return to Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="btnExit">
            <property name="text">
             <string>End</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageListAttendance">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="3" column="2" colspan="2">
           <widget class="QPushButton" name="btnListAttendanceBack">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="rdAdd">
            <property name="text">
             <string>Add Attendance</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <widget class="QPushButton" name="btnListAttendanceNext">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Depending on which radio button you have selected, you will go to a different place in the program.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="rdView">
            <property name="text">
             <string>&amp;View Attendance</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QListWidget" name="lstListEvents"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageAddAttendance">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="2" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="btnAddAttendanceToggle">
            <property name="text">
             <string>24 Hour Toggle</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="3">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="6">
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
          <item row="3" column="4">
           <widget class="QPushButton" name="btnAddAttendanceSave">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QPushButton" name="btnAddAttendanceBack">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Select Available Times</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageViewAttendance">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="6">
           <widget class="QPushButton" name="btnViewAttendanceQuit">
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="btnViewAttendanceReturn">
            <property name="text">
             <string>Return to Start</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="btnViewAttendanceBack">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnViewAttendanceToggle">
            <property name="text">
             <string>24-Hour Toggle</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="6">
           <widget class="QTableWidget" name="tableWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
